framework/src/actions/MaterialOutputAction.C

```diff
@@ -189,7 +190,14 @@ MaterialOutputAction::act()
                    " to restrict the material properties to output");
       _problem->addAuxVariable("MooseVariableConstMonomial", var_name, params);
     }
-    if (material_names.size() > 0)
+
+    bool print_aux_creation = true;
+    const auto common_actions = _app.actionWarehouse().getActions<CommonOutputAction>();
+    mooseAssert(common_actions.size() <= 1, "Should not be more than one CommonOutputAction");
+    const Action * common = common_actions.empty() ? nullptr : *common_actions.begin();
+    if (common && !common->getParam<bool>("print_automatic_aux_variable_creation"))
+      print_aux_creation = false;
+    if (material_names.size() > 0 && print_aux_creation)
```

Comment:
so you are really adding this to silence it?

Reply:
I mean the PR in general not just these few lines

Reply:
Oh ha! Yes, I'll change it...